be/src/runtime/minidump.cpp

```diff
@@ -32,11 +32,12 @@ int Minidump::_signo = SIGUSR1;
 std::unique_ptr<google_breakpad::ExceptionHandler> Minidump::_error_handler = nullptr;
 
 // Save the absolute path and create_time of a minidump file
-struct FileStat {
+struct MinidumpFileStat {
     std::string abs_path;
```

Comment:
Maybe we could remove minidump， it is not too much useful any more.

Reply:
I agree,
Usually online clusters do not enable core dump, not sure if minidump will help

Reply:
There is already a core stack in be.out, most of us use this core stack.

Reply:
> Maybe we could remove minidump， it is not too much useful any more.

i agree too

Reply:
> > Maybe we could remove minidump， it is not too much useful any more.
> 
> i agree too

ok, I plan to remove minidump.